VA-Explorer/va_explorer
# Cod config

This PR does three things: 
1. moves algorithm settings to .env file (defaulting to hardcoded values when not set in environment)
2. adds logic to run_coding_algorithms enabling you to re-code all VAs by overriding existing CODs (set `--overwrite=True`)
3. when `overwrite=True`, exports old CODs to a csv for safekeeping

To test:
1. run coding algorithms normally - `./manage.py run_coding_algorithms`
2. Change algorithm settings in env file (e.g. `INTERVA_MALARIA=l`)
3. re-run coding algorithms command with override=True - `./manage.py run_coding_algorithms --overwrite True`
4. Confirm old CODs are saved to csv
5. (optional) compare old CODs to new CODs and check if settings made a difference
